crates/solidity/outputs/npm/tests/src/doc-examples/using-queries.ts

```diff
@@ -24,7 +24,7 @@ test("using queries", async () => {
     const cursor = parse_output.createTreeCursor();
```

Comment:
I also believe that the user docs are still using the old "bindings" terminology.